models/spleen_deepedit_annotation/scripts/transforms.py

```diff
@@ -0,0 +1,38 @@
+from typing import Dict
+
+import numpy as np
+from einops import rearrange
+from monai.transforms.transform import Transform
+
+
+class OrientationGuidanceMultipleLabelDeepEditd(Transform):
+    def __init__(self, ref_image="image", label_names=None):
+        """
+        Convert the guidance to the RAS orientation
+        """
+        self.ref_image = ref_image
+        self.label_names = label_names
+
+    def transform_points(self, point, affine):
+        """transform point to the coordinates of the transformed image
+        point: numpy array [bs, N, 3]
+        """
+        bs, n = point.shape[:2]
```

Comment:
another different place, using lowercase "n" can help to avoid flake8 issue
cc @AndreasHeumann

Reply:
Perfectly fine!